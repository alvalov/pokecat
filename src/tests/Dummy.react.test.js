import React from "react";
import {shallow} from "enzyme";
import jestRenderer from "react-test-renderer";
import {Dummy} from "../features/dummy/Dummy"; 

test("Component should render text if prop is true", ()=> {
    const component = jestRenderer.create(
        <Dummy shouldRenderText={true} text="Hello World!"/>
    );
    const tree=component.toJSON();
    expect(tree).toMatchSnapshot();
})

test("Shallow render works", ()=>{
    const component = shallow(
        <Dummy shouldRenderText={true} text="Hello World!"/>
    );
    expect(component.text()).toEqual("Hello World!");
})