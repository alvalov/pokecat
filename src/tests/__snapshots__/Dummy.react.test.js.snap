// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should render text if prop is true 1`] = `
<div>
  Hello World!
</div>
`;
